second-edition/src/ch07-02-controlling-visibility-with-pub.md

```diff
@@ -130,15 +201,23 @@ Hooray! We have a different error! Yes, different error messages are a cause
 for celebration. The new error shows `` function `connect` is private ``, so
 let’s edit *src/client.rs* to make `client::connect` public too:
 
+Opa! Temos um erro diferente! Sim, diferentes mensagens de erro são uma causa
+para a celebração. O novo erro mostra que `` function `connect` is private ``(`` função `connect` é privada ``), então
+vamos editar *src/client.rs* para torná-la públicao `client::connect` também:
```

Comment:
Sugestão: "Sim, **mensagens diferentes** de erro são **motivo para comemorar**. O novo erro mostra que **a função `connect` é privada (``function `connect` is private``)**, então vamos editar *src/client.rs* para torná-la **pública** ~`client::connect`~ também:"